users/postgres/users.go

```diff
@@ -81,7 +81,7 @@ func (ur userRepository) Update(ctx context.Context, user users.User) error {
 }
 
 func (ur userRepository) UpdateUser(ctx context.Context, user users.User) error {
-	q := `UPDATE users SET metadata = :metadata WHERE email = :email AND status = 'enabled'`
+	q := `UPDATE users SET id = :id, metadata = :metadata WHERE email = :email AND status = 'enabled'`
```

Comment:
This should be removed after to implement this feature: https://github.com/MainfluxLabs/mainflux/issues/234